sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -1309,76 +1317,153 @@
+            "page": "",
+            "left": "",
+            "top": "",
+            "width": "",
+            "height": ""
+          },
+          "type": "hidden",
           "input": true,
+          "tableView": false,
           "placeholder": "",
           "prefix": "",
-          "customClass": "",
           "suffix": "",
           "multiple": false,
-          "protected": false,
           "unique": false,
-          "persistent": true,
           "hidden": false,
           "clearOnHide": true,
           "refreshOn": "",
-          "redrawOn": "",
-          "modalEdit": false,
-          "dataGridLabel": true,
-          "labelPosition": "right",
+          "dataGridLabel": false,
+          "labelPosition": "top",
           "description": "",
           "errorLabel": "",
           "tooltip": "",
           "hideLabel": false,
           "tabindex": "",
           "disabled": false,
           "autofocus": false,
+          "widget": {
+            "type": "input"
+          },
+          "validateOn": "change",
+          "validate": {
+            "required": false,
+            "custom": "",
+            "customPrivate": false,
+            "strictDateValidation": false,
+            "multiple": false,
+            "unique": false
+          },
+          "conditional": {
+            "show": null,
+            "when": null,
+            "eq": ""
+          },
+          "allowCalculateOverride": false,
+          "showCharCount": false,
+          "showWordCount": false,
+          "allowMultipleMasks": false,
+          "inputType": "hidden",
+          "id": "esb4nx"
+        },
+        {
+          "label": "My offering is not listed",
+          "description": "",
+          "tooltip": "",
+          "shortcut": "",
+          "inputType": "checkbox",
+          "customClass": "",
+          "tabindex": "",
+          "hidden": false,
+          "hideLabel": false,
+          "autofocus": false,
+          "disabled": false,
+          "tableView": false,
+          "modalEdit": false,
+          "defaultValue": false,
+          "persistent": true,
+          "protected": false,
           "dbIndex": false,
+          "encrypted": false,
+          "redrawOn": "",
+          "clearOnHide": true,
           "customDefaultValue": "",
-          "calculateValue": "",
+          "calculateValue": "if (data.offeringnotListed) {\r\n  data.selectedOfferingDate = \"\";\r\n}",
           "calculateServer": false,
-          "widget": null,
-          "attributes": {},
-          "validateOn": "change",
+          "allowCalculateOverride": false,
           "validate": {
             "required": false,
+            "customMessage": "",
             "custom": "",
             "customPrivate": false,
+            "json": "",
             "strictDateValidation": false,
             "multiple": false,
             "unique": false
           },
+          "errorLabel": "",
+          "key": "offeringnotListed",

```

Comment:
Please adjust the case to offeringNotListed.

Reply:
If this is the only change to form definition, please ignore it.
I noticed that the form has a lot of other similar case issues. Not a problem for this PR.

Reply:
renamed

Reply:
yes, its form definition change